scripts/periodic_cleanup.py

```diff
@@ -188,7 +188,7 @@ def cleanup_maas_entities(self):
 
             # sort agents with most recently connected agent last
             agents = sorted(
-                (a for a in self.agents if a.id == entity.agent_id),
+                [a for a in self.agents if a.id == entity.agent_id],
```

Comment:
This doesn't need to be changed, sorted only does one pass and the generator can't be accessed outside the function.

Reply:
Yeah, sorted returns a list, but I changed it for consistency.